app/templates/gulpfile.js

```diff
@@ -257,13 +258,13 @@ gulp.task('optimize', ['inject', 'test'], function() {
 
   var assets = $.useref.assets({searchPath: './'});
   // Filters are named for the gulp-useref path
-  var cssFilter = $.filter('**/*.css');
-  var jsAppFilter = $.filter('**/' + config.optimized.app);
-  var jslibFilter = $.filter('**/' + config.optimized.lib);
+  var cssFilter = $.filter('**/*.css', {restore: true});
+  var jsAppFilter = $.filter('**/' + config.optimized.app, {restore: true});
+  var jslibFilter = $.filter('**/' + config.optimized.lib, {restore: true});
 
```

Comment:
Looks good according to https://www.npmjs.com/package/gulp-filter#restoring-filtered-files. Weird that this never gave issues before..